reorg security service naming (#715)
aerogear/mobile-docs: modules/ROOT/pages/device-security.adoc

:context: security
ifdef::context[:parent-context: {context}]

[id='device-checks']

> = Adding {device-security-service} to your mobile app

Reviewer: ```suggestion
= Implementing {device-security-service} in your mobile app
```
